core/ui/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/core/ui/exercise/ExerciseActionButtons.kt

```diff
@@ -211,22 +216,22 @@ class BoundExerciseActions(
 fun InfoMessageCard() {
```

Comment:
The MessageCard is currently also shown for Quizes, but quiz participation is possible through the app